group_project_v2/stage_components.py

```diff
@@ -112,10 +112,11 @@ class GroupProjectVideoResourceXBlock(BaseGroupProjectResourceXBlock):
     CATEGORY = "gp-v2-video-resource"
     STUDIO_LABEL = _(u"Video Resource")
     PROJECT_NAVIGATOR_VIEW_TEMPLATE = 'templates/html/components/video_resource.html'
+    BRIGHTCOVE_ACCOUNT_ID = '6057949416001'
```

Comment:
This should not be hardcoded. Make it a configuration item that can be set via XBlocks settings.

Reply:
done